Add more explicit ECS/EC2 DogStatsD instructions for endpoint (#16474)
DataDog/documentation: content/en/containers/amazon_ecs/_index.md

]
```

In addition to this port mapping, set the environment variable `DD_DOGSTATSD_NON_LOCAL_TRAFFIC` to `true`.

> This setup allows the DogStatsD traffic to route from the application containers, through the host and host port, to the Datadog Agent container. However, this setup requires the application container to use the host's private IP Address for this traffic. This can be done by setting the enviroment variable `DD_AGENT_HOST` to to the private IP address of the EC2 Instance fetched from the Instance Metadata Service (IMDS). Alternatively this can be set in the code during initialization. This strategy for DogStatsD is the same as for APM, you can see the [APM documentation for examples of setting the Agent endpoint][17].

Reviewer: ```suggestion
This setup allows the DogStatsD traffic to route from the application containers, through the host and host port, to the Datadog Agent container. However, this setup requires the application container to use the host's private IP address for this traffic. This can be done by setting the environment variable `DD_AGENT_HOST` to the private IP address of the EC2 instance fetched from the Instance Metadata Service (IMDS). Alternatively, this can be set in the code during initialization. The implementation for DogStatsD is the same as for APM, see [configure the trace agent endpoint][17] for examples of setting the Agent endpoint.
```

> This setup allows the DogStatsD traffic to route from the application containers, through the host and host port, to the Datadog Agent container. However, this setup requires the application container to use the host's private IP address for this traffic. This can be done by setting the environment variable `DD_AGENT_HOST` to the private IP address of the EC2 instance fetched from the Instance Metadata Service (IMDS). Alternatively, this can be set in the code during initialization. The implementation for DogStatsD is the same as for APM, see [configure the trace agent endpoint][17] for examples of setting the Agent endpoint.

Reviewer: ```suggestion
This setup allows the DogStatsD traffic to be routed from the application containers, through the host and host port, to the Datadog Agent container. However, the application container must use the host's private IP address for this traffic. This can be enabled by setting the environment variable `DD_AGENT_HOST` to the private IP address of the EC2 instance, which can be retrieved from the Instance Metadata Service (IMDS). Alternatively, this can be set in the code during initialization. The implementation for DogStatsD is the same as for APM, see [configure the Trace Agent endpoint][17] for examples of setting the Agent endpoint.
```

> For APM and DogStatsD double check the security group settings on your EC2 instances. Make sure these ports are not open to the public.

Reviewer: ```suggestion
Ensure that the security group settings on your EC2 instances do not publicly expose the ports for APM and DogStatsD.
```
